app/controllers/planning_applications/review/heads_of_terms_controller.rb

```diff
@@ -3,26 +3,15 @@
 module PlanningApplications
   module Review
     class HeadsOfTermsController < BaseController
-      def show
-        respond_to do |format|
-          format.html
-        end
-      end
-
-      def edit
-        respond_to do |format|
-          format.html
-        end
-      end
-
       def update
         respond_to do |format|
           format.html do
             if heads_of_term.update(review_params)
               redirect_to planning_application_review_tasks_path(@planning_application),
                 notice: I18n.t("review.heads_of_terms.update.success")
             else
-              render :show
```

Comment:
could we apply same pattern?

change the 

```ruby 
def heads_of_term
  @planning_application.heads_of_term
end
```
to 

```ruby 
def set_heads_of_term
  @heads_of_term = @planning_application.heads_of_term
end
```

happy to do it as additional PR if you want

so this would be
```ruby
flash.now[:alert] = @heads_of_term.errors.messages.values.flatten.join(", ")
```

Reply:
I don't really like the pattern of using a method to set an instance variable and then using the instance variable. I think it's preferable to use a method call where possible.

It makes sense for variables that will be used in a view, because instance variables are needed for that, methods aren't accessible from the view. But for cases like this I don't think it makes sense.